pkg/dockerregistry/server/repositorymiddleware.go

```diff
@@ -102,6 +112,8 @@ type repository struct {
 	// if true, the repository will check remote references in the image stream to support pulling "through"
 	// from a remote repository
 	pullthrough bool
+	// acceptschema2 allows to refuse the manifest schema version 2
+	acceptschema2 bool
```

Comment:
This is something I'd like to be able to override using just an environment variable. And I think it is necessary for later painless transition to schema v2. Could you please define and parse something like `REGISTRY_MIDDLEWARE_REPOSITORY_OPENSHIFT_ACCEPTSCHEMA2`?